packages/chart/src/components/LinearChart.tsx

```diff
@@ -100,6 +100,7 @@ const LinearChart = forwardRef<SVGAElement, LinearChartProps>(({ parentHeight, p
   const [animatedChart, setAnimatedChart] = useState(false)
   const [point, setPoint] = useState({ x: 0, y: 0 })
   const [suffixWidth, setSuffixWidth] = useState(0)
+  const [autoPadding, setAutoPadding] = useState(0)
```

Comment:
I hate to make this name longer, but I wonder if we should explicitly state what type of autoPadding this is (ie. yAxisAutoPadding) should we introduce other automatic padding into the mix.